Parameters.java

```diff
@@ -7,6 +7,21 @@
 
 public class Parameters {
 
+	public enum AlphabetType {
+		NUCLEOTIDES("AGTC"),
+		AMINO_ACIDS("ARNDBCEQZGHILKMFPSTWYV");
+
```

Comment:
Don't you want a close brace here?

Reply:
The close brace is on line 23 instead of 13, because we need to include a constructor. Otherwise, we can't pass in valid characters.

Reply:
I see-- this is a fancy enum.